benchmarks/README.md

```diff
@@ -0,0 +1,78 @@
+# Benchmarks for Optuna
+
+Interested in measuring Optuna's performance? 
+You are very perceptive. 
+Under this directory, you will find scripts that we have prepared to measure Optuna's performance.
+
+In this document, we explain how we measure the performance of Optuna using the scripts in this directory.
+The contents of this document are organized as follows.
+- [Performance Benchmarks with `kurobako`](#performance-benchmarks-with-kurobako)
+
+## Performance Benchmarks with `kurobako`
+
+We measure the performance of black-box optimization algorithms in Optuna with 
+[`kurobako`](https://github.com/optuna/kurobako) using `benchmarks/run_kurobako.py`.
+You can manually run this script on the GitHub Actions if you have a write access on the repository.
+Or, you can locally execute the `benchmarks/run_kurobako.py`.
+We explain both of method here.
+
+### How to Run on the GitHub Actions
```

Comment:
Please follow this instruction to try the benchmarking. Note that until this PR is merged, you need to temporarily change the default branch of the forked Optuna repository to "add-performance-bench".